scripts/module/stadium.nut

```diff
@@ -0,0 +1,25 @@
+// Stadium billboards
+
+currentModelOffset <- 0;
+BillboardsBegin <- 2042;
+
+function onScriptLoad() {
+  billboard <- CreateObject(BillboardsBegin, 1, -1037.08, 1340.258, 36.552, 255);
+}
+
+function onPlayerCommand( player, cmd, text ) {
+  if (! player.Admin ) {
+    return;
+  }
+  cmd = cmd.tolower();
+  if (cmd == "sbb" || cmd == "swapbillboard") {
+    MessagePlayer("Swapped stadium billboard.", player);
+    billboard.Delete();
+    currentModelOffset++;
+    if (currentModelOffset == 3) {
+      currentModelOffset = 0;
+    }
+    billboard <- CreateObject(BillboardsBegin + currentModelOffset, player.World,
+                              -1037.08, 1340.258, 36.552, 255);
```

Comment:
You probably want to move this model switching in a "SetStadiumBillboard" or something already, so it will be easier to export it in the future - basically specify an API asap.